### PR TITLE
Implementation of Error events

### PR DESCRIPTION
**Pull request by CTI777**
Related to https://github.com/Activiti/Activiti/issues/2743
Depends on:
https://github.com/Activiti/activiti-cloud-api/pull/117
https://github.com/Activiti/Activiti/pull/2908
https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/376

### CODE REVIEW

**Comment by CTI777** [comment]
Do we need here any extra tests?

File: activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/conf/AuditJPAAutoConfiguration.java [hunk 1]
```diff
@@ -269,20 +270,7 @@ public TimerRetriesDecrementedEventConverter timerRetriesDecrementedEventConvert
     
     @ConditionalOnMissingBean
     @Bean
-    public MessageReceivedEventConverter messageReceivedEventConverter(EventContextInfoAppender eventContextInfoAppender) {
```
Comment (ffazzini) [inline activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/conf/AuditJPAAutoConfiguration.java]: why do not we need the converters related to messages anymore?

File: activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/converters/ErrorReceivedEventConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    public void should_convert_toEntityErrorReceivedEvent() {
+        CloudBPMNErrorReceivedEventImpl event = createErrorReceivedEvent();
+
+        ErrorReceivedAuditEventEntity auditEventEntity = (ErrorReceivedAuditEventEntity) eventConverter.convertToEntity(event);
+
+        assertThatIsEqualTo(auditEventEntity, event);
+    }
+    
+    @Test
+    public void shoudld_convert_toAPIErrorReceivedEvent() {
```
Comment (ffazzini) [inline activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/converters/ErrorReceivedEventConverterTest.java]: typo "should", and extra underscore between "convert" and "toAPIErrorReceivedEvent"

Comment (CTI777) [reply activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/converters/ErrorReceivedEventConverterTest.java]: Name is changed

File: activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/conf/AuditJPAAutoConfiguration.java (hunk 1 above)
Comment (CTI777) [reply activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/conf/AuditJPAAutoConfiguration.java]: Thank you for this tip (wrongly removed)... Fixed now!

**Review by ffazzini — APPROVED** [review]

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
You should cover the new event with an integration test as well. Similar to this one https://github.com/Activiti/activiti-cloud-audit-service/blob/develop/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java#L512.

**Comment by CTI777** [comment]
A test is added.

**Review by erdemedeiros — APPROVED** [review]

**Pull request merged**